src/lib/toDate.js

```diff
@@ -2,6 +2,6 @@ import assertString from './util/assertString';
 
 export default function toDate(date) {
```

Comment:
I've added this refactor to #2075, thanks for the suggestion!

Reply:
Okay, after looking at the README again I've decided to undo this change in #2075 
Apparently `toDate` is being exported and the docs mention `convert the input string to a date, or `null` if the input is not a date.`
Therefore this would be a breaking change. I thought `toDate` was just an internal function, but alas not.

Reply:
Good catch, thanks. But it really makes me wonder why this decision was made. I mean, converting the type to `null` is a bit pointless, since the native Date object already handles invalid dates. So, IMHO, if someone calls a function `toDate` they should be able to expect a `Date` type. WDYT?

Reply:
Reverted